src/main/java/de/triology/cas/services/LogoutUriEnabledRegexRegisteredService.java

```diff
@@ -0,0 +1,17 @@
+package de.triology.cas.services;
+
+import java.net.URI;
+
+public class LogoutUriEnabledRegexRegisteredService extends org.jasig.cas.services.RegexRegisteredService {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS2160)